### PR TITLE
Disable cypress tests for real

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
Missed a job in https://github.com/mattermost/mattermost-plugin-apps/pull/492

#### Ticket Link
None

### CODE REVIEW

**Comment by hanzei** [comment]
@toninis Gentle reminder to review the PR :smiley:

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -175,7 +175,7 @@ jobs:
   e2e-cypress-tests-pinned: # Run only on master push and scheduled runs
     runs-on: ubuntu-latest-4-cores
     # Disabled broken test
-    if: ${{ false || github.ref_name == 'master' || github.event_name == 'schedule' }} 
+    if: ${{ false && github.ref_name == 'master' || github.event_name == 'schedule' }}
```
Comment (toninis) [inline .github/workflows/ci.yaml:178]:
```suggestion
    if: ${{ false && ( github.ref_name == 'master' || github.event_name == 'schedule' ) }} 
```

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -302,7 +302,8 @@ jobs:
 
   e2e-cypress-tests-master: # Run only on master push and scheduled runs
     runs-on: ubuntu-latest-4-cores
-    if: ${{ github.ref_name == 'master' || github.event_name == 'schedule' }}
+    # Disabled broken test
+    if: ${{ false && github.ref_name == 'master' || github.event_name == 'schedule' }}
```
Comment (toninis) [inline .github/workflows/ci.yaml:306]:
```suggestion
    if: ${{ false && ( github.ref_name == 'master' || github.event_name == 'schedule' ) }} 
```

**Review by toninis — CHANGES_REQUESTED** [review]

**Comment by hanzei** [comment]
@toninis Good catch :+1:

**Review by toninis — APPROVED** [review]

**Pull request merged**